charts/redpanda-console-oauth2proxy/values.yaml

```
@@ -92,7 +93,11 @@ console:
 
 # Configuration for the oauth2proxy sidecar container.
 oauth2Proxy:
-  image: quay.io/oauth2-proxy/oauth2-proxy:v7.5.1
+  image:
+    registry: quay.io/
```

```suggestion
    registry: quay.io
```
The `/` should be handled by the chart